### ISSUE TITLE
a question about dataloader

### ISSUE THREAD

**Issue by ShellingFord221** [yunjey/pytorch-tutorial#154]
Hi
    If I don't want to test the model in batch (i.e. test the test data one by one), should my code be:
    
    test_loader = torch.utils.data.DataLoader(dataset=test_dataset,
                                          batch_size=batch_size,                             # this line still remains
                                          shuffle=False)
    ...
    for images, labels in test_loader:        # not for batch_idx, (images, labels) in enumerate(test_loader):
    ...

And other codes don't change. Am I correct? Thanks!

**Comment by weixia1** [comment]
u can set **batch_size =1** ,if you want test the data one by one
`batch_size = 1
test_loader = torch.utils.data.DataLoader(dataset=test_dataset,
                                      batch_size=batch_size,                             # this line still remains
                                      shuffle=False)`
